### PR TITLE
feat/club events' schema

### PR DESCRIPTION
**Pull request by KambojRajan**

### CODE REVIEW

File: server/db/schema/clubs.schema.ts [hunk 1]
```diff
@@ -43,6 +53,7 @@ export const clubs = pgTable('clubs', {
 export const clubsRelations = relations(clubs, ({ many, one }) => ({
   clubMembers: many(clubMembers),
   clubSocials: many(clubSocials),
+  clubEvents: many(events),
```
Comment (GetPsyched) [inline server/db/schema/clubs.schema.ts:56]:
```suggestion
  clubEvents: many(events),
  clubMembers: many(clubMembers),
  clubSocials: many(clubSocials),
```
This is alphabetically ordered

File: server/db/schema/events.schema.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import { clubs, persons } from '.';
+
+export const events = pgTable('events', {
+  id: serial('id').primaryKey(),
+  name: varchar('name', { length: 128 }).notNull(),
+  description: varchar('description', { length: 512 }),
+  startDate: timestamp('date').notNull(),
+  endDate: timestamp('date').notNull(),
+  clubId: integer('club_id')
```
Comment (GetPsyched) [inline server/db/schema/events.schema.ts:18]: If this is a event specific to a club, name the table `club_events`

File: server/db/schema/index.ts [hunk 3]
```diff
@@ -20,3 +20,4 @@ export * from './sponsored-research-projects.schema';
 export * from './staff.schema';
 export * from './student-academic-details.schema';
 export * from './students.schema';
+export * from './events.schema';
```
Comment (GetPsyched) [inline server/db/schema/index.ts:23]: This file is also alphabetically ordered. And in light of the previous comment, this would be `club-events.schema`

**Review by GetPsyched — CHANGES_REQUESTED** [review]

File: server/db/schema/events.schema.ts (hunk 2 above)
Comment (KambojRajan) [reply server/db/schema/events.schema.ts:18]: An event may or may not be related to a club, such as a convocation

Comment (GetPsyched) [reply server/db/schema/events.schema.ts:18]: Then make the `club_id` nullable.

File: server/db/schema/clubs.schema.ts [hunk 4]
```diff
@@ -27,6 +27,11 @@ export const clubs = pgTable('clubs', {
   tagline: varchar('tagline', { length: 256 }).notNull(),
   email: varchar('email', { length: 256 }).notNull(),
   aboutUs: varchar('about_us').notNull(),
+  howToJoinUs: varchar('how_to_join_us').notNull(),
+  whyToJoinUs: varchar('why_to_join_us').notNull(),
+  notifications: varchar('notifications')
+    .array()
+    .default(sql`'{}'`),
```
Comment (GetPsyched) [inline server/db/schema/clubs.schema.ts:34]: Do this in a different PR please
